app/controllers/api/v1/projects_controller.rb

```diff
@@ -1,7 +1,8 @@
 class Api::V1::ProjectsController < ApplicationController
   before_action :find_project, only: [:edit, :update, :destroy]
   def index
-    @projects = Project.all
+    @projects = @current_user.projects.all
```

Comment:
we already have current_user method in application controller, should we call current_user method instead of instance variable @current_user ?